pom.xml

```diff
@@ -46,6 +46,14 @@
                     <linkXRef>false</linkXRef>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>16</source>
+                    <target>16</target>
+                </configuration>
+            </plugin>
```

Comment:
this shouldn't be necessary, if you have any problem don't hesitate to contact us in the platform's chat